### PR TITLE
MF-832 Invalid Content-Type in MQTT Version Endpoint, MF-822 Return 404 for non `/version` paths on MQTT adapter

### LINKED ISSUE #822: Return 404 for non `/version` paths on MQTT adapter
**BUG REPORT**

1. What were you trying to achieve?
Get version endpoint with misspelled `/verision` path.

2. What are the expected results?
Status code 404 Not Found.

3. What are the received results?
Status code 204 No Content.

4. What are the steps to reproduce the issue?
Send an HTTP request to MQTT service path other than `/version`

5. In what environment did you encounter the issue?
I'm running latest docker composition.

6. Additional information you deem important:
This does not affect service functionality (therefore this is the low priority issue), but it is not consistent with other services.

### LINKED ISSUE #832: Invalid Content-Type in MQTT Version Endpoint
In the following code, a JSON is returned from the version endpoint, but it has `text/plain` for Content-Type.

https://github.com/mainflux/mainflux/blob/a8836d67e0fa78e4a111b8b63557ca90396ab40b/mqtt/aedes/mqtt.js#L117

I think this cause issue in clients for decoding the results.

### PR DESCRIPTION
**Pull request by 1995parham**
### What does this do?
It fixes content-type and not-found issues in MQTT version endpoint. if you use invalid route instead of getting empty response you get 404 error with a not-found message.

### Which issue(s) does this PR fix/relate to?
Resolve #832 
Resolve #822 

### List any changes that modify/break current functionality
It doesn't break or change current functionality.

### Have you included tests for your changes?
No

### Did you document any new/modified functionality?
No

### CODE REVIEW

File: mqtt/aedes/mqtt.js [hunk 1]
```diff
@@ -112,11 +112,13 @@ esclient.on('error', function(err) {
 function startWs() {
     var server = http.createServer();
     server.on('request', (req, res) => {
+        res.setHeader('Content-Type', 'application/json');
         if (req.url === '/version') {
             res.statusCode = 200;
-            res.setHeader('Content-Type', 'text/plain; charset=utf-8');
             res.end(`{"service":"mqtt-adapter","version":"${version}"}`);
```
Comment (nmarcetic) [inline mqtt/aedes/mqtt.js:118]: We don't use template literals (e.g mixing with ES6). Just a tip, but we should consider refactoring and using ES6.

**Review by nmarcetic — APPROVED** [review]

**Review by drasko — APPROVED** [review]
LGTM

**Comment by drasko** [comment]
Please change issue name to something like `MF-832 - <PR title>`

**Comment by 1995parham** [comment]
@drasko I have changed the title to issues that this pull request resolves.

**Pull request merged**